sasmodels/gengauss.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+from __future__ import division, print_function
+
+import sys
+
+import numpy as np
+from numpy.polynomial.legendre import leggauss
+
+def gengauss(n, path):
```

Comment:
cache the results of leggauss?

Reply:
gengauss.py produces a C program file containing the values and weights for gaussian quadrature of size n.  generate.py checks if such a file exists prior to calling gengauss, so it is cached.  This happens in generate.set_integration_size, which for now is only available from sascomp for model testing.